functions/mqhandlers.go

```diff
@@ -573,3 +590,25 @@ func handleFwUpdate(server string, payload *models.FwUpdate) {
 	}
 
 }
+
+// MQTT Fallback Mechanism
+func MQFallback(MQFallbackStop chan bool) {
+	for {
+		select {
+		case <-MQFallbackStop:
+			// Stop the goroutine
+			MQFallbackRunning = false
+			return
+		default:
+			MQFallbackRunning = true
+
+			// Call http netclient config pull
+			_, err := Pull(false)
+			if err != nil {
+				slog.Error("failed to pull")
+			}
+
+			time.Sleep(time.Second * 30) // Execute pull every 30 seconds
```

Comment:
use a time.Ticker

Reply:
I thought so. But @abhishek9686 wants this to handle both mq connection dropout situation and mq peer updates lost situation.